parser-typechecker/src/Unison/CommandLine/InputPatterns.hs

```diff
@@ -963,17 +971,18 @@ links = InputPattern
   )
 
 unlink :: InputPattern
-unlink = InputPattern "unlink" ["delete.link"]
-  [(Required, exactDefinitionQueryArg),
-   (Required, exactDefinitionQueryArg) ]
-  "`unlink src dest` removes a link from `src` to `dest`."
+unlink = InputPattern
+  "unlink"
+  ["delete.link"]
+  [(Required, exactDefinitionQueryArg), (OnePlus, exactDefinitionQueryArg)]
+  "`unlink dest src` removes a link to `dest` from `src`."
```

Comment:
Should update the help here (like `link`) to explain the multiple metadata option.